components/sidebar/sidebar.jsx

```diff
@@ -35,7 +35,6 @@ export default class Sidebar extends Component {
           height={ availableHeight } />
 
         <div className="sidebar__inner">
-          <img src="https://img.shields.io/badge/webpack-v2.3.3-blue.svg" />
```

Comment:
Not sure if we should entirely remove it from here without having it somewhere else that's consistently displayed. This came up a few times a while back which is why we added it to the sidebar for lack of a better place. @bebraw @TheLarkInn what do you think?